README.md

```diff
@@ -7,11 +7,11 @@ in [data-plane-api](https://github.com/envoyproxy/data-plane-api). It started li
 [go-control-plane](https://github.com/envoyproxy/go-control-plane), but building an idiomatic Java implementation is
 prioritized over exact interface parity with the Go implementation.
 
-Both v2 and v3 resources as well as transport versions are supported. Migrating
-to v3 is recommended as Envoy will drop v2 support at EOY 2020 (see
+Only v3 resources as well as transport versions are now supported. Migrating
```

Comment:
I think we should drop any references to migrating to v2 in the readmes, since with this patch we are effectively removing the code that made those migration paths possible.

Reply:
There is link to documentation one in the next line. I've fixed link right now